aten/src/ATen/core/jit_type.h

```diff
@@ -2247,6 +2256,14 @@ struct TORCH_API ClassType : public NamedType {
   std::vector<torch::jit::Function*> methods_;
   std::vector<torch::jit::Function*> staticmethods_;
 
+  // Map of overloaded methods associated with this class that are keyed by
+  // method name
+  std::unordered_map<std::string, std::vector<torch::jit::Function*>>
+      overloaded_methods_;
+
+  // Map of mangled name to function pointer
+  std::unordered_map<std::string, torch::jit::Function*> mangled_to_function_;
```

Comment:
i think the mangled names should be stored normally as all the other functions

Reply:
Yeah this is just to keep track of what names maps to what instead of having to go through entire list of methods